pkg/templates/kubeadm/v1beta3/kubeadm.go

```diff
@@ -119,9 +120,15 @@ func NewConfig(s *state.State, host kubeoneapi.HostConfig) ([]runtime.Object, er
 			Kind:       "ClusterConfiguration",
 		},
 		Networking: kubeadmv1beta3.Networking{
-			PodSubnet:     cluster.ClusterNetwork.PodSubnet,
-			ServiceSubnet: cluster.ClusterNetwork.ServiceSubnet,
-			DNSDomain:     cluster.ClusterNetwork.ServiceDomainName,
+			PodSubnet: join(
+				cluster.ClusterNetwork.PodSubnet,
+				cluster.ClusterNetwork.PodSubnetIPv6,
+			),
+			ServiceSubnet: join(
+				cluster.ClusterNetwork.ServiceSubnet,
+				cluster.ClusterNetwork.ServiceSubnetIPv6,
+			),
```

Comment:
Doesn't the order depend on if IPv4+IPv6 or IPv6+IPv4 is used?

Reply:
oh yes :grimacing: